android-components/components/browser/engine-system/src/main/res/values-sk/strings.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@

Reply:
No, this won't work, because these changes will be reverted on the next import. In general, you shouldn't modify localized files in this repository.

This is not necessarily issue to fix, filed https://github.com/mozilla-l10n/android-l10n/issues/645

Reply:
That makes sense. Thanks for catching that! I won't change strings.xml then. It would be great if https://github.com/mozilla-l10n/android-l10n/issues/645 can (eventually) result in license notices being added; otherwise, we can exclude these files from the license linter.